pkg/daemon/ceph/rgw/objectstore.go

```diff
@@ -219,6 +219,21 @@ func deletePools(context *Context, lastStore bool) error {
 		}
 	}
 
+	// Delete erasure code profile if any
+	erasureCodes, err := ceph.ListErasureCodeProfiles(context.context, context.ClusterName)
+	if err != nil {
+		return fmt.Errorf("failed to list erasure code profiles for cluster %s: %+v", context.ClusterName, err)
+	}
+	objectStoreErasureCode := fmt.Sprintf("%s_ecprofile", context.Name)
```

Comment:
is there a way that we can make this naming format in a unified place with https://github.com/rook/rook/blob/master/pkg/daemon/ceph/client/erasure-code-profile.go#L104?

i had to search around to figure out how ec profiles are named in the first place, it'd be good for this to all be in one place.

Reply:
Sounds good. Good point. I will do something here to unify it.